### PR TITLE
fix(slider): limit slider value

### PR DESCRIPTION
**Pull request by danielkaxis**
Limits the slider value to min and max values
limit slider value

### CODE REVIEW

File: packages/core/src/Slider/index.tsx [hunk 1]
```diff
@@ -340,7 +340,17 @@ export const Slider: React.FC<SliderProps> = ({
     }
   }, [tickConfig])
 
-  const fraction = (value - min) / (max - min)
+  const fraction = useMemo(() => {
+    if (value > max) {
+      return (max - min) / (max - min)
```
Comment (Tigge) [inline packages/core/src/Slider/index.tsx:345]: Always 1.0?

File: packages/core/src/Slider/index.tsx [hunk 2]
```diff
@@ -340,7 +340,17 @@ export const Slider: React.FC<SliderProps> = ({
...
+    }
+
+    if (value < min) {
+      return (min - min) / (max - min)
```
Comment (Tigge) [inline packages/core/src/Slider/index.tsx:349]: Always 0? `min - min = 0`

File: packages/core/src/Slider/index.tsx [hunk 3]
```diff
@@ -340,7 +340,17 @@ export const Slider: React.FC<SliderProps> = ({
     }
   }, [tickConfig])
 
-  const fraction = (value - min) / (max - min)
```
Comment (Tigge) [inline packages/core/src/Slider/index.tsx:343]: Perhaps just `Math.max(0, Math.min(1, (value - min) / (max - min)))`, or use the `clamp` function in this file already. `clamp((value - min) / (max - min))`?

**Review by Tigge — CHANGES_REQUESTED** [review]

File: packages/core/src/Slider/index.tsx (hunk 1 above)
Comment (danielkaxis) [reply packages/core/src/Slider/index.tsx]: Opps, yes :smile:

File: packages/core/src/Slider/index.tsx (hunk 2 above)
Comment (danielkaxis) [reply packages/core/src/Slider/index.tsx]: Opps, yes :smile:

File: packages/core/src/Slider/index.tsx (hunk 3 above)
Comment (danielkaxis) [reply packages/core/src/Slider/index.tsx:343]: Done

**Review by Tigge — APPROVED** [review]

**Pull request merged**